authz/providers/azure/rbac/rbac_test.go

```diff
@@ -195,9 +195,13 @@ func TestLogin(t *testing.T) {
 		if !time.Now().Before(u.expiresAt) {
 			t.Errorf("Expiry not set properly. Expected it to be after the current time. Actual: %v", u.expiresAt)
 		}
+
+		// Normalize to second precision for comparison
+		expectedExpiresOn := expiresOn.Add(-tokenExpiryDelta).Truncate(time.Second)
+		actualExpires := u.expiresAt.Truncate(time.Second)
 		// Expect u.expires to be tokenExpiryDelta before expiresOn
```

Comment:
this comment is not relevant anymore this comment is not relevant anymore? or might be confusing given that you're normalizing it above for comparison